hartsfield/models/user.py

```diff
@@ -84,23 +84,13 @@ def to_api_json(self):
 
     @classmethod
     def _load_user(cls, uid=None):
-        calnet_profile = {}  # TODO: get_calnet_user_for_uid(app, uid) if uid else {}
-        expired = False  # TODO: calnet_profile.get('isExpiredPerLdap', True)
-        is_admin = False
-        is_teaching = False
-        is_active = (is_teaching or is_admin) and not expired
+        is_active = True if uid else False
```

Comment:
Ripley code authorizes people based on role, but Hartsfield should treat anyone with a found UID as active. (People who aren't active won't make it into a Flask-Login session.)